### ISSUE TITLE
Feature: additional configuration with `ignore-case: true`

### ISSUE THREAD

**Issue by SimonGolms** [azat-io/eslint-plugin-perfectionist#78]
### Describe the rule

I am currently using your `eslint-plugin-perfectionist` in my project and I appreciate the functionality it provides. However, I have noticed that for all rules `ignore-case` is set to `false` by default.

This default setting is not ideal for my use case, as it requires me to list each rule individually in order to change the `ignore-case` setting to `true`. This process is quite tedious and maybe I miss a new rule in the future since I cannot sticky with any of the recommended configurations.

Therefore, I kindly request to consider some more configurations with `ignore-case: true` as the default setting. This change would greatly simplify the configuration process and enhance the usability of your plugin.

### Code example

Currently:
```js
// .eslintrc.js
module.exports = {
  rules: {
    // ...
    "perfectionist/sort-imports": [
      "error",
      {
        "ignore-case": true,
        type: "natural",
      },
    ],
    "perfectionist/sort-named-imports": [
      "error",
      {
        "ignore-case": true,
        type: "natural",
      },
    ],
    "perfectionist/sort-objects": [
      "error",
      {
        "ignore-case": true,
        type: "natural",
      },
    ]
    // ...
  }
}
```

Expected
```js
// .eslintrc.js
module.exports = {
  extends: [
    // ...
    "plugin:perfectionist/recommended-natural-ignore-case",
  ],
}
```

### Additional comments

Personally I see the ignorance of the upper/lower case as much more natural as in the case of the pure alphabetical sorting. Maybe one solution could be to update `recommended-natural` configuration with `ignore-case: true`?

https://github.com/azat-io/eslint-plugin-perfectionist/blob/45947068318de150191cdcaa89fd45690cf797b7/index.ts#L137-L141

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
I'm thinking of exporting a function to create a plugin config generation. This will be possible for those using flat config (which will become standard in the next version of ESLint).

Like this:

```js
// eslint.config.js
import { createPerfectionistConfig } from 'eslint-plugin-perfectionist'

export default [
  createPerfectionistConfig({
     type: 'natural', 
     order: 'asc', 
     'ignore-case': false, 
  })
]
```

**Comment by azat-io** [comment]
Thank you for your issue!

We decided not to create a large number of configs. But I think you're right. In most cases, users are more accustomed to searching for values in alphabetical order and ignore case.

For this reason, we have made the ignoreCase field enabled by default.

This feature was implemented in 63efbf7a2a637e69b461e574b8d3e195c625f9ad and released in [v3.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.0.0).

If you'd like, you can support the release with a retweet:
https://x.com/azat_io_en/status/1815367279191761054
